style.css

```diff
@@ -407,14 +423,118 @@ form > button:hover {
   height: 70px;
 }
 
+.portfolio {
+  background-image: url('../images/headline_section_bg.png');
+  background-repeat: no-repeat;
+  background-size: cover;
+  background-position: center center;
+  display: flex;
+  justify-content: center;
+}
+
+.header-container {
+  padding: 20px var(--containers-side-padding);
+}
+
+.project-body {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
+.project-body > * {
+  width: calc(var(--wrapper-width) - 50px);
+}
+
+.project-picture {
+  display: flex;
```

Comment:
**Optional Changes**
- [ ] The CSS selector proper `display` with values `flex` is over repeatedly used. This defiles the principle of `DRY`  which adds Code Smell. Let us find time and refactor these.

Reply:
Thank you, I will make the necessary changes.